dev-requirements-py3.txt

```diff
@@ -1,7 +1,7 @@
 flake8==3.8.3
 bandit==1.6.2
 demisto-py==2.0.22
-git+git://github.com/demisto/demisto-sdk@finally-fixed-3.9
+git+git://github.com/demisto/demisto-sdk@master
```

Comment:
as to our latest discussion on the matter, better to use a fixed commit hash instead of a branch name.